response_operations_ui/views/messages.py

```diff
@@ -106,6 +106,12 @@ def view_conversation(thread_id):
     category = thread_conversation["category"]
     refined_thread = [_refine(message, category) for message in reversed(thread_conversation["messages"])]
     latest_message = refined_thread[-1]
+
+    # In some cases the survey_id is added to the technical and misc messages. This causes rerouting to the survey inbox
+    # By removing the survey_id from the message thread, we are redirected back to either technical or misc inboxes
```

Comment:
While this may resolve the issue I guess the question is whether the `survey_id` can be omitted to technical and misc messages in the first place?

Reply:
This was an erroneous decision based on poor data. It has been removed.